app/src/main/java/com/git/amarradi/leafpad/NoteEditActivity.java

```diff
@@ -61,7 +61,6 @@ protected void onCreate(Bundle savedInstanceState) {
             String type = intent.getType();
             if (Intent.ACTION_SEND.equals(action) && type != null) {
                 if (type.startsWith("text/plain")) {
-                    Note.makeId();
```

Comment:
This call creates a new UUID but does not assign it somewhere. Nothing is done with it. I guess even the compiler will drop that line. So my suggestion would be to drop it here.

Reply:
Leafpad had the problem that it did not display new notes that were sent to leafpad via send to. See #31 The notes were saved in the sharedprefs but were not displayed. The debugger showed that the ID remained null. This seemed to be fixed with this call. Now it works as expected.

Reply:
Leafpad only shows Notes with ID != Null

Reply:
`Note.makeId()` is a pure function with no side effects. It just creates an UUID that never gets stored somewhere. So it has no effect anywhere. 

But I understand  your fears. Would it be possible to test that special case? 

To make it even more tempting to remove  that line:

I bet either:
* a good bottle of red wine,
* a six pack of Guinness
* or a big cup of ice 

that nothing will change if you remove this line.

What about that?

Reply:
Yes, I understand, but why does sending to leafpad work with this function?

Reply:
Now I have tested it again. I don't understand why this line is there. leafpad had the problem that until version 1.8 the sent messages were not saved and the ID=null.

Reply:
So we'll enjoy a six pack of Guinness together and accept this change?